tools/topology/topology2/cavs-src-mixin-mixout-hda.conf

```diff
@@ -73,7 +73,7 @@ Object.Pipeline {
 		}
 	]
 
-	passthrough-be [
+	passthrough-playback-be [
```

Comment:
this is wrong @plbossart. Like I mentioned passthrough-be is not only for playback. If you look at the direction below, it is meant for capture. So changing this to playback is wrong

Reply:
It's all needlessly complicated. No wonder no one can figure this out.